playbooks/molecule/resources/xnat/verify.yml

```diff
@@ -16,3 +16,63 @@
           - response.status == 200
           - response.server == "nginx"
           - response.content is search('MIRSG XNAT')
+
+- name: Verify correct Docker version is installed
+  hosts: xnat_cserv
+  tasks:
+    - name: Get Docker server version
+      ansible.builtin.command:
+        cmd: "{% raw %}docker version -f '{{.Server.Version}}'{% endraw %}"
+      changed_when: false
+      register: server_version
+
+    - name: Get Docker client version
+      ansible.builtin.command:
+        cmd: "{% raw %}docker version -f '{{.Client.Version}}'{% endraw %}"
+      changed_when: false
+      register: client_version
+
+    - name: Check Docker server version
+      ansible.builtin.assert:
+        that:
+          - server_version.stdout is version( docker_version )
+          - client_version.stdout is version( docker_version )
```

Comment:
this will fail if `docker_version` is `latest`. Perhaps once the issue with XNAT is fixed we should remove the `docker_version` variable and stop pinning the version?

Reply:
or at least stop checking the version here

Reply:
I like the idea of pinning personally